fusedrug/utils/cluster/cluster_using_mmseqs.py

```diff
@@ -222,14 +239,31 @@ def cluster(
 cluster_impl = cluster  # for backward compatibility
 
 
-def _run_system_cmd(cmd: str) -> None:
-    print("about to run: ", cmd)
-    res = subprocess.run(cmd, shell=True, check=False, capture_output=True)
-    if len(res.stdout) > 0:
+def _run_system_cmd(cmd: str, capture_output: bool = True) -> None:
+    print(f"about to run: {cmd}")
+    res = subprocess.run(cmd, shell=True, check=False, capture_output=capture_output)
+    if res.stdout and len(res.stdout) > 0:
         print("stdout=")
         print(res.stdout.decode())
-    if len(res.stderr) > 0:
+    if res.stderr and len(res.stderr) > 0:
         print("stderr=")
         print(res.stderr.decode())
-    if res.returncode != 0:
+    if res.returncode and res.returncode != 0:
```

Comment:
can you describe this line modification motivation ?

Reply:
Just for future reference:

Double checked that running twice the script :
```python
import subprocess

res = subprocess.run("echo 1", shell=True, check=False, capture_output=?)

print(res.stdout)
print(res.stderr)
print(res.returncode)
```

With `capture_output=True` we get:
```
b'1\n'
b''
0
```

And `capture_output=False` gives us:
```
1
None
None
0
```